catkit/emulators/thorlabs/FW102C.py

```diff
@@ -0,0 +1,50 @@
+from abc import ABC, abstractmethod
+
+import pyvisa
+
+import catkit.hardware.thorlabs.ThorlabsFW102C
+
+
+class FW102CEmulator(ABC):
+    """ Emulates pyvisa specifically for the FW102C. """
+
+    Commands = catkit.hardware.thorlabs.ThorlabsFW102C.ThorlabsFW102C.Commands
+
+    class constants:
+        class StatusCode:
+            success = pyvisa.constants.StatusCode.success
+
+    def __init__(self, initial_position=1):
+        self.last_status = self.constants.StatusCode.success
+        self.position = initial_position
+
+    def ResourceManager(self, *args, **kwargs):
+        return self
+
+    def open_resource(self, *args, **kwargs):
+        return self
+
+    def close(self):
+        pass
+
+    @abstractmethod
+    def move_filter(self, position):
+        """ Implement to make filter changes for the simulated system, e.g., a Poppy model. """
+        pass
```

Comment:
This gets implemented in hicat (in https://github.com/spacetelescope/hicat-package/pull/539).